PencilCase/PencilCase/www/index.html

```diff
@@ -32,7 +32,7 @@
     <!-- Make sure the page id here is exactly the same as it defined in Consts.Pages -->
     <div data-role="page" id="HomePage">
         <!-- ko with: homePage() -->
-        <div data-role="header" data-position="fixed">
+        <div data-role="header" data-position="fixed" data-tap-toggle="false">
```

Comment:
https://stackoverflow.com/questions/5763619/jquery-mobile-fixed-toolbar-and-footer-bar-disappears
https://forum.jquery.com/topic/fixed-header-disappears-on-click-within-textbox